src/plugins/navigation/public/side_navigation/index.tsx

```diff
@@ -8,34 +8,34 @@
  */
 
 import { css } from '@emotion/react';
-import { euiThemeVars } from '@kbn/ui-theme';
 import React, { Suspense, type FC } from 'react';
-import { EuiSkeletonRectangle } from '@elastic/eui';
+import { EuiSkeletonRectangle, useEuiTheme } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 
 import type { Props as NavigationProps } from './side_navigation';
 
 const SideNavComponentLazy = React.lazy(() => import('./side_navigation'));
 
-export const SideNavComponent: FC<NavigationProps> = (props) => (
-  <Suspense
-    fallback={
-      <EuiSkeletonRectangle
-        css={css`
-          margin: ${euiThemeVars.euiSize};
-        `}
-        width={16}
-        height={16}
-        borderRadius="s"
-        contentAriaLabel={i18n.translate(
-          'navigation.sideNavigation.loadingSolutionNavigationLabel',
-          {
-            defaultMessage: 'Loading solution navigation',
-          }
-        )}
-      />
-    }
-  >
-    <SideNavComponentLazy {...props} />
-  </Suspense>
-);
+export const SideNavComponent: FC<NavigationProps> = (props) => {
+  const { euiTheme } = useEuiTheme();
+  return (
+    <Suspense
+      fallback={
+        <EuiSkeletonRectangle
+          css={css`
+            margin: ${euiTheme.size.base};
+          `}
+          width={16}
+          height={16}
+          borderRadius="s"
+          contentAriaLabel={i18n.translate(
+            'navigation.sideNavigation.loadingSolutionNavigationLabel',
+            { defaultMessage: 'Loading solution navigation' }
+          )}
+        />
+      }
+    >
+      <SideNavComponentLazy {...props} />
+    </Suspense>
+  );
```

Comment:
This just affects the **loading state** of the solution side nav. You will need to throttle the browser's network or play back a video frame-by-frame to see it.

https://github.com/user-attachments/assets/7714583e-f041-4318-9f8d-f4f7565b521f